wave_utils/tests/utils/matlab_scripts/test_jacobian_p7_p7_Composition_wrt_p2.m

```diff
@@ -0,0 +1,36 @@
+clear;
```

Comment:
Do matlab scripts belong? @navganti

Reply:
So @jskhu will double check that the underlying functionality has been ported over to C++ unit tests. For the ones that haven't been ported over to C++ I think it's fine to keep them.